regressionTests/testjs/src/index.ts

```diff
@@ -0,0 +1,262 @@
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+
+    // -- Node group
+
+    $("#LoadHeartBeat").click(async function () {
+        try {
+            let response = await nodeGroup.handleHeartbeat();
+            displayTestsSuites("LoadHeartBeatOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    // -- Transaction group
+
+    $("#LoadSendTransaction").click(async function () {
+        try {
+            let response = await transactionGroup.handleSendTransaction();
+            displayTestsSuites("LoadSendTransactionOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    $("#LoadSendMultipleTransactions").click(async function () {
+        try {
+            let response = await transactionGroup.handleSendMultipleTransactions();
+            displayTestsSuites("LoadSendMultipleTransactionsOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    // -- Validator group
+
+    $("#LoadValidatorStatistics").click(async function () {
+        try {
+            let response = await validatorGroup.handleValidatorStatistics();
+            displayTestsSuites("LoadValidatorStatisticsOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+
+    // -- VM Values group
+
+    $("#LoadVmValuesQuery").click(async function () {
+        try {
+            let response = await vmValuesGroup.handleVmValuesQuery();
+            displayTestsSuites("LoadVmValuesQueryOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    $("#LoadVmValuesInt").click(async function () {
+        try {
+            let response = await vmValuesGroup.handleVmValuesInt();
+            displayTestsSuites("LoadVmValuesIntOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    $("#LoadVmValuesString").click(async function () {
+        try {
+            let response = await vmValuesGroup.handleVmValuesString();
+            displayTestsSuites("LoadVmValuesStringOutput", response);
+        } catch (error) {
+            onError(error);
+        }
+    });
+
+    // - Run all tests
+    $("#RunAllTests").click(async function () {
+        await runAllTests();
+    })
+});
+
+function onError(error: Error) {
+    let html = Err.html(error);
+    $("#ErrorModal .error-text").html(html);
+    $("#ErrorModal").modal("show");
+}
+
+function getProxyUrl(): string {
+    return $("#ProxyInput").val();
+}
+
+function getPrivateKey(): string {
+    return $("#PrivateKeyInput").val().trim();
+}
+
+async function runAllTests() {
+    $("#LoadAccountDataShouldWork").click();
+    $("#LoadAccountDataShouldErr").click();
+    $("#LoadAccountNonce").click();
+    $("#LoadAccountBalance").click();
+    $("#LoadAccountShard").click();
+    //$("#LoadAccountTransactions").click();
```

Comment:
Write a comment to explain why they are commented out (because they broadcast transactions).

Reply:
added